### PR TITLE
feat(twitter): Implement additional twitter tags

### PR DESCRIPTION
**Pull request by ags1773**
https://github.com/quintype/page-builder/issues/2072

### CODE REVIEW

File: src/author-tags.js [hunk 1]
```diff
@@ -7,13 +7,28 @@ import { get } from "lodash";
...
+function getTwitterCreator(storyData) {
+  const twitterData = get(storyData, ["data", "story", "authors", 0, "social", "twitter"], {});
+  const twitterUrl = twitterData.url || "";
+  const twitterHandle = twitterData.handle || "";
+  if (twitterHandle.startsWith("@")) return twitterHandle;
+  const twitterHandleFromUrl = twitterUrl.split("/").pop();
+  if (twitterHandleFromUrl) {
+    if (twitterHandleFromUrl.startsWith("@")) return twitterHandleFromUrl;
+    return `@${twitterHandleFromUrl}`;
+  }
```
Comment (ReenaSingh07) [inline src/author-tags.js:32]: can we add this as separate util function as its used in another place as well

File: test/author_tags_test.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+                  url: "https://twitter.com/JohnDoe123",
+                  handle: "",
+                },
+              },
+              name: "John Doe",
+            },
+          ],
+        },
+      },
+    };
```
Comment (ReenaSingh07) [inline test/author_tags_test.js:52]: can we define this in one place ?

**Review by ReenaSingh07 — CHANGES_REQUESTED** [review]

File: src/author-tags.js (hunk 1 above)
Comment (ags1773) [reply src/author-tags.js:32]: There isn't much code reuse. Also it will be more legible to have it this way

File: test/author_tags_test.js (hunk 2 above)
Comment (ags1773) [reply test/author_tags_test.js:52]: They're all different

**Review by ReenaSingh07 — APPROVED** [review]

**Review by snitin4260 — APPROVED** [review]

**Review by Jeevan-Kishore — APPROVED** [review]

**Pull request merged**